<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QDialog" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidgetConfig">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxComPort">
           <property name="title">
            <string>COM Port</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QComboBox" name="comboBoxComPort"/>
            </item>
            <item>
             <widget class="QPushButton" name="buttonSaveConfig">
              <property name="text">
               <string>Зберегти</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/icons/Save.ico</normaloff>:/icons/Save.ico</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="buttonConfigAdmin">
           <property name="text">
            <string>Конфігурація</string>
           </property>
           <property name="icon">
            <iconset resource="AC.qrc">
             <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonOpenCsvFile">
           <property name="text">
            <string>Відкрити CSV-файл</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/icon_csv.ico</normaloff>:/icons/icon_csv.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCancelConfig">
           <property name="text">
            <string>Вихід</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/exit.ico</normaloff>:/icons/exit.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBoxCalibr">
           <property name="title">
            <string>Калібровка каналів</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,10">
              <item>
               <widget class="QLabel" name="labelCalibrChannel">
                <property name="text">
                 <string>Канал</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxCalibrChannels"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,10">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="buttonCalibr">
                <property name="text">
                 <string>Калібрувати</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConfigAdminBack">
           <property name="text">
            <string> Вихід / Назад </string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icons/exit.ico</normaloff>:/icons/exit.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="AC.qrc"/>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
